### PR TITLE
Add Pod Disruption Budget term to glossary

### PR DESCRIPTION
**Pull request by McCoyAle**
Add glossary entries for Pod Disruption Budget term in reference to #5993

### CODE REVIEW

**Comment by McCoyAle** [comment]
I did sign the account, but have 2 accounts so I signed the CLA for the second. As I believe something may have went wrong when I committed my changes.

File: content/en/docs/reference/glossary/pod-disruption-budget.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ - PDB
+related:
+ - pod
+ - container
+tags:
+ - operation
+short-description: >
+ An object that limits the number of Pods of a replicated application, that are down simultaneously from voluntary disruptions.
+long-description: >
+ A [Pod Disruption Budget](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#pod-priority) allows an application owner to create an object for a replicated application, that ensures a certain number or percentage of Pods with an  assisgned label will not be voluntarily evicted at any point in time. PDBs cannot prevent an involuntary disruption, but will count against the budget.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/pod-disruption-budget.md]: is there a reason to not link to this page?
https://kubernetes.io/docs/concepts/workloads/pods/disruptions/

Comment (neolit123) [reply content/en/docs/reference/glossary/pod-disruption-budget.md]: `assisgned` typo.

Comment (McCoyAle) [reply content/en/docs/reference/glossary/pod-disruption-budget.md]: I have update the link.

**Pull request closed without merging**